README.md

```diff
@@ -167,22 +146,108 @@ You can also make a package request by creating a [new issue](https://github.com
 
 You can also [create a package yourself](https://github.com/luxtorpeda-dev/packages/blob/master/docs/Creating_a_Package.md)
 
-## Development
+## Other installation methods
+
+### Pre-requisites
+
+Compatibility tool will work on any modern 64-bit Linux distribution.
+All packaged games are compiled for Steam Runtime Sniper environment and any additional dependencies are provided in the download.
+
+**Using Luxtorpeda with [Steam native runtime](https://wiki.archlinux.org/index.php/Steam/Troubleshooting#Steam_native_runtime) may or may not work, but is not supported.**
+
+### Installation (using tarball)
+
+*The packages.json for the supported packages and getting the latest versions will get updated on each launch of luxtorpeda, without any input needed from the user. New releases of the luxtorpeda client will need to be downloaded manually, but a new release of the client is not required when a package is created or updated, unless that package depends on a new feature (which will be noted in the release notes).*
+
```

Comment:
Same for this, this is a general detail about luxtorpeda that helps people understand how it works, not really about a particular install method.